### PR TITLE
Incorporate new data from RECOVERY trial

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

@Jeff-Field pointed out that the RECOVERY trial released data on TCZ last week - I did a quick write-up on this!
He also pointed out that we were using a verb that is not a verb, so I replaced all of those!

### Suggested reviewers (optional)

@SiminaB 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -657,7 +656,16 @@ Although these estimates are similar, it is important to note that they are draw
...
+Of the 21,550 patients enrolled in the RECOVERY trial at the time, 4,116 adults hospitalized with COVID-19 across the 131 sites in the United Kingdom were assigned to the arm of the trial evaluating the effect of TCZ.
+Among them, 2,022 were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
+The primary outcome measured was 28-day mortality, and TCZ was found to reduce 28-day mortality from 33% of patients receiving SOC alone to 29% of those receiving TCZ, corresponding to a rate ratio of 0.86 (95% CI 0.77-0.96; _p_ = 0.007).
+TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group ( (rate ratio 1.22, 95% CI 1.12-1.34, _p_ < 0.0001).
+A potential association between the TCZ administration in combination with a corticosteroid was observed, but the authors note that caution is advisable in light of the number of statistical tests conducted.
```
Comment (SiminaB) [inline content/20.pharmaceuticals.md:664]: This is a bit unclear - a potential association between TCZ administration in combination with a corticosteroid and _what_? It look like something is missing.

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -667,7 +675,8 @@ Reactivation of hepatitis B and herpes simplex virus 1 was also reported in a sm
...
 An increased risk of bacterial infection was also identified in a systematic review of the literature, based on the unadjusted estimates reported [@doi:10.2139/ssrn.3642653].
-TCZ administration to COVID-19 patients is not without risks, may introduce additional risk of developing secondary infections, and should be approached especially cautiously for patients who have latent viral infections.
+In the RECOVERY trial, however, only three out of 2,022 participants in the group receiving TCZ developed adverse reactions determined to be associated with the intervention, and no excess deaths were reported [@doi:10.1101/2021.02.11.21249258].
+TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; therefore, it should be approached especially cautiously for patients who have latent viral infections.
```
Comment (SiminaB) [inline content/20.pharmaceuticals.md:679]: It seems like the risks are not super high though, based on the available data?

**Review by SiminaB — COMMENTED** [review]
Generally looks very good! I only have a couple of clarification questions.

File: content/20.pharmaceuticals.md (hunk 1 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:672]:
```suggestion
A potential statistical interaction between TCZ and corticosteroids was observed, with the combination providing greater mortality benefits than TCZ alone, but the authors note that caution is advisable in light of the number of statistical tests conducted.
```

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:687]:
```suggestion
TCZ administration to COVID-19 patients is not without risks and may introduce additional risk of developing secondary infections; however, while caution may be prudent when treating patients who have latent viral infections, the results of the RECOVERY trial indicate that adverse reactions to TCZ are very rare among COVID-19 patients broadly.
```

**Review by rando2 — COMMENTED** [review]
Thank you so much @SiminaB!

**Comment by rando2** [comment]
@SiminaB if these changes looked OK to you, would you mind approving so it can be merged? Thank you so much for the feedback!!!

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -665,7 +664,16 @@ Although these estimates are similar, it is important to note that they are draw
...
 
-Additionally, there are possible risks associated with the administration of TCZ for COVID-19.
+On February 11, 2021, a preprint describing the first randomized control trial of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
+Of the 21,550 patients enrolled in the RECOVERY trial at the time, 4,116 adults hospitalized with COVID-19 across the 131 sites in the United Kingdom were assigned to the arm of the trial evaluating the effect of TCZ.
+Among them, 2,022 were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
+The primary outcome measured was 28-day mortality, and TCZ was found to reduce 28-day mortality from 33% of patients receiving SOC alone to 29% of those receiving TCZ, corresponding to a rate ratio of 0.86 (95% CI 0.77-0.96; _p_ = 0.007).
+TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group ( (rate ratio 1.22, 95% CI 1.12-1.34, _p_ < 0.0001).
```
Comment (agitter) [inline content/20.pharmaceuticals.md:671]:
```suggestion
TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group (rate ratio 1.22, 95% CI 1.12-1.34, _p_ < 0.0001).
```

**Review by agitter — COMMENTED** [review]
I can approve this but found one typo first.  Also have the two suggestions to address @SiminaB's comments been applied?

**Comment by rando2** [comment]
Hi @agitter, I just committed everything! I assume it's usually easier for reviewers if I leave the responses active, but in this case it makes sense to commit them!

**Review by agitter — APPROVED** [review]
Looks good to me.  Thanks for the TCZ updates.

**Pull request merged**